src/index.tsx

```diff
@@ -1,7 +1,15 @@
 import ReactDOM from 'react-dom';
 import { App } from './App';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { TodoModal } from './components/TodoModal/TodoModal';
+import { TodoList } from './components/TodoList/TodoList';
+import { Loader } from './components/Loader/Loader';
 
 ReactDOM.render(
   <App />,
+  <TodoFilter />,
+  <TodoModal />,
+  <TodoList />,
+  <Loader />,
```

Comment:
Why are you doing all this imports in this file? We don`t need to change this index file. Use original files structure.